libs/cardano-protocol-tpraos/testlib/Test/Cardano/Protocol/Binary/Cddl.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
```

Comment:
It is usually a bad idea to disable this warning. Is there a reason why we can't delete redundant constraints?
```suggestion
```